peachjam/admin.py

```diff
@@ -1138,6 +1155,15 @@ def save(self, *args, **kwargs):
 
         return self.instance
 
+    def convert_to_paragraphs(self, value):
+        if not value:
+            return ""
+
+        if isinstance(value, list):
+            return "".join([f"{item}\n" for item in value])
```

Comment:
```suggestion
            return "\n".join(value)
```

i think that's sufficient?